### DISCUSSION TITLE
Apparent motion blur when tweening between two PCams

### DISCUSSION THREAD

**Discussion by Zorigami** [ramokz/phantom-camera#133] [Question / Help Wanted]
First of all, thanks so much for your work on this plugin!
My current issue is that I have two PCams (PCam1 and PCam2.) PCam 1 is offset 300px above the player, and PCam2 is offset 300px below. When my player is falling and matches certain conditions, I'm switching between the two cameras, but as the tween moves the Camera2D between them, there's a lot of motion blur on the player. I've been wracking my brain for solutions for a while, and figured I'd ask here.

**Comment by ramokz** [comment]
That's an interesting one.

I'm not too sure what would be causing the motion blur. There's nothing in the plugin that explicitly applies an effect like that.

This might be unrelated, but have noticed that a falling target with a `PCam` that follows it will gradually drift away from it - almost as if the camera can't keep up with the falling character. Which is a bug, though not a significant one.

To that point, does the motion blur only occur when the character is falling, and is it only during the tween, or does it continue even after it's finished tweening between `PCam1` and `Pcam2`?

**Comment by ZenithStar** [comment]
Are you sure there's actually motion blur? The Godot rendering pipeline does not include any motion blur. https://docs.godotengine.org/en/stable/tutorials/3d/environment_and_post_processing.html

Try taking a screenshot of your game when you think it's blurring and see if it's actually blurred. It's most likely your eyes playing tricks on you.
